doc/source/basic-use.rst

```diff
@@ -1,153 +1,259 @@
+With a release downloaded, iocage is now able to create jails. There are
+five types of jails, three basic and two advanced. Basic jail types are
+**full**, **clone**, and **base** jails. Advanced jails types are
+**empty** and **templates**, but these are discussed in the
+:ref:`Jail Types` section, along with more detailed descriptions of the
+basic jail types.
+
+Depending on the user's requirements, the :command:`create` subcommand
+can be tweaked to create any of the three basic types. By default,
+iocage creates a fully independent jail of the current host's RELEASE,
+setting the TAG property to the current date.
+
+The :command:`create` command can be used to quickly create a basic
+jail:
+
+:samp:`# iocage create -r 11.0-RELEASE`
 
-Creating a jail is real simple, just run:
+This basic jail is fully independent.
 
-``iocage create``
+To create a full jails with multiple clones, type:
 
-This will create a fully independent jail.
+:samp:`# iocage create -r 11.0-RELEASE -c 2`
 
-To create a lightweight jail (clone) run:
+The numeric value after the **-c** flag is used to designate the number
+of clone jails to create. In the above example, *two* jails are created,
+**one** *full* jail and **one** clone jail.
 
-``iocage create -c``
+A simple basejail is created with the **-b** flag:
 
-To create a base jail:
+:samp:`# iocage create -b -r 11.0-RELEASE`
 
-``iocage create -b``
+After designating the type of jail to create with the option flags,
+specific jail **properties** can also be set. For example:
 
-To create a jail and set its IP address and tag property run:
+:samp:`# iocage create -r 11.0-RELEASE tag=myjail boot=on`
 
-``iocage create -c tag=myjail ip4_addr="em0|10.1.1.10/24"``
+Creates a FreeBSD 11.0 jail with the custom tag *myjail* and sets the
+jail to start at system boot time.
 
-For more information please read iocage(8).
+More information about iocage jail properties is available in the
+iocage(8) FreeBSD manual page, which is accessed on a FreeBSD system by
+typing :command:`man iocage`.
 
-Listing jails
+.. index:: Listing Jails
+.. _Listing Jails:
+
+Listing Jails
 -------------
 
-To list all jails run:
+To list all jails:
+
+:samp:`# iocage list`
+
+To see all downloaded RELEASEs:
+
+:samp:`# iocage list -r`
+
+View available templates with:
 
-``iocage list``
+:samp:`# iocage list -t`
 
-To see all downloaded RELEASEs run:
+.. index:: Jail start stop restart
+.. _Start Stop Restart Jail:
 
-``iocage list -r``
+Start, Stop, or Restart a Jail
+------------------------------
 
-To see available templates run:
+Jails can be started, stopped, or restarted at any time with iocage. By
+default, new jails are in a *down* (stopped) state. To see the status of
+all jails, use :command:`iocage list` and read the **STATE** column.
 
-``iocage list -t``
+Use each jail's UUID or custom TAG to start, stop, or restart it. When
```

Comment:
s/TAG/name/ The partial stuff should now apply to both name and UUID, so it should be reworded as Partial entries are acceptable.